ruleset.xml

```diff
@@ -6,10 +6,11 @@
     <exclude-pattern>./vendor/*</exclude-pattern>
     <exclude-pattern>*/thirdparty/*</exclude-pattern>
 
-    <!-- Show progress and output sniff names on violation, and add colours -->
+    <!-- Show progress and output sniff names on violation, add colours, and run in parallel mode -->
     <arg value="p" />
     <arg name="colors" />
     <arg value="s" />
+    <arg name="parallel" value="10" />
```

Comment:
Having a read of https://github.com/squizlabs/PHP_CodeSniffer/issues/1732

I think this change is complety fine, this comment sums it up.
> make sure PHP is compiled with PCNTL support, or the CLI setting wont do anything.

So setting for 10, should be fine as well as @Cambis outlined.


I dislike the reduction of per file output to singular parallel reporting.
So until they add that output back the output isn't helpful, so we should quieten it

```suggestion
    <!-- Show progress and output sniff names on violation, add colours, and run in parallel mode -->
    <arg value="q" />
    <arg name="colors" />
    <arg value="s" />
    <arg name="parallel" value="10" />
```

Reply:
Using `q` shows no progress at all right? I think its better to have some indication of progress instead of none at all